<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1500</width>
    <height>1000</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:rgb(194, 194, 194);</string>
  </property>
  <widget class="QPushButton" name="backbutton">
   <property name="geometry">
    <rect>
     <x>1254</x>
     <y>922</y>
     <width>211</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size:25pt;
background-color: rgb(0, 85, 127);</string>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
  <widget class="QPushButton" name="imageButton1">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>30</y>
     <width>491</width>
     <height>381</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="imageButton2">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>30</y>
     <width>491</width>
     <height>381</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="imageButton3">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>490</y>
     <width>491</width>
     <height>381</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="imageButton4">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>490</y>
     <width>491</width>
     <height>381</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="nextButton">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>942</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 85, 127);
font-size: 20pt;</string>
   </property>
   <property name="text">
    <string>다음</string>
   </property>
  </widget>
  <widget class="QPushButton" name="beforeButton">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>942</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 85, 127);
font-size: 20pt;</string>
   </property>
   <property name="text">
    <string>이전</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
